postqa/jsonshim.py

```diff
@@ -0,0 +1,40 @@
+"""Tools for shimming the JSON output from validate_drp into the simpler
+JSON schema expected by the SQuaSH dashboard's POST /api/jobs/ endpoint.
+"""
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+from builtins import *  # NOQA
+from future.standard_library import install_aliases
+install_aliases()  # NOQA
+
+
+def shim_validate_drp(vdrp_json,
+                      accepted_metrics=('PA1', 'AM1', 'AM2', 'AM3')):
+    """Convert JSON structure from validate DRP into the JSON expected by
+    SQuaSH's POST /api/jobs/ endpoint.
+
+    This makes a full job document, but only populates the ``measurements``
+    field.
+    """
```

Comment:
I would suggest adding a comment that this code isn't currently validating the input format.